__tests__/app.test.js

```diff
@@ -76,6 +75,46 @@ describe("app", () => {
         });
     });
   });
-});
+  describe("/api/articles/:article_id", () => {
+    it("200 - GET: respond with an article object with the expected properties", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then((response) => {
+          const articleArrObj = response.body.article;
 
-//writing somehting new to test git
+          articleArrObj.forEach((articleObj) => {
+            expect(articleObj).toMatchObject({
+              author: expect.any(String),
+              article_id: expect.any(Number),
```

Comment:
Can we just amend this to expect the specific article_id please (in this case 1)?